src/contacts/uwsgi-conf.ini

```diff
@@ -0,0 +1,10 @@
+[uwsgi]
+uid = root
+gid = root
+
+processes = 4
+master = true
+module = contacts:create_app()
+die-on-term = true
+
+disable-logging=True
```

Comment:
Note - this logging only refers to the uwsgi access logs, which happen on every request and were noisy. And I couldn't find a way to match them to the user-provided log level, as we do with the flask app logs. So I turned them off for now. 

But the Flask logs are intact / unchanged, still respond to log level.